kombu/transport/virtual/exchange.py

```diff
@@ -11,6 +11,7 @@
 from kombu.utils import escape_regex
 
 import re
+import collections
```

Comment:
Used a `collections.OrderedDict` because it is already in use throughout the project (for example in `kombu/connection.py`) even if it excludes python 2.6 from the compatible interpreters.